pydm/widgets/pushbutton.py

```diff
@@ -312,51 +316,53 @@ def validate_password(self):
     def alarm_severity_changed(self, new_alarm_severity):
         """
         Callback invoked when the Channel alarm severity is changed.
-        This callback is not processed if the widget has no channel associated with it.
-        This callback handles the composition of the stylesheet to be applied and the call
-        to update to redraw the widget with the needed changes for the new state.
+
+        This callback is not processed if the widget has no channel associated
+        with it. It handles the composition of the stylesheet to be applied and
+        the call to update to redraw the widget with the needed changes for the
+        new state.
 
         Parameters
         ----------
         new_alarm_severity : int
-            The new severity where 0 = NO_ALARM, 1 = MINOR, 2 = MAJOR and 3 = INVALID
+            New severity: 0 = NO_ALARM, 1 = MINOR, 2 = MAJOR and 3 = INVALID
         """
         if self._alarm_sensitive_content:
-            self._style = dict(self.alarm_style_sheet_map[self.ALARM_CONTENT][new_alarm_severity])
+            self._style = dict(self.alarm_style_sheet_map[self.ALARM_CONTENT][
+                                                        new_alarm_severity])
             style = compose_stylesheet(style=self._style, obj=self)
             self.setStyleSheet(style)
             self.update()
 
     @pyqtSlot()
     def sendValue(self):
         """
-        Send a new value to the channel
+        Send a new value to the channel.
 
         This function interprets the settings of the PyDMPushButton and sends
         the appropriate value out through the :attr:`.send_value_signal`.
         """
-        if not self._pressValue or self.value is None:
+        if self._pressValue is None or self.value is None:
```

Comment:
And here is the actual bug correction, checking if  `_pressValue` is None.
If the string in `_pressValue` cannot be converted by `channelType` an `ValueError` will be raised. Should this error be handled?